boards/arm/nrf5340dk_nrf5340/nrf5340dk_nrf5340_cpunet_defconfig

```diff
@@ -4,6 +4,9 @@ CONFIG_SOC_SERIES_NRF53X=y
 CONFIG_SOC_NRF5340_CPUNET_QKAA=y
 CONFIG_BOARD_NRF5340DK_NRF5340_CPUNET=y
 
+# Disable PM, not supported on net core
+CONFIG_PM=n
```

Comment:
Could agree on the patch, if there could at least be a way to achieve the desired behavior, while not having to undefine something at board_defconfig

Reply:
Can you recommend one?

Reply:
Yes. You could have a Kconfig capability, e.g. config CPU_SUPPORTS_PM, and add a dependency, so you won't need to set to =n here.

Reply:
@ioannisg That'd be #31722 which is stalled because it's not a great solution either, since virtually every board would have to set it *and* it isn't clear what it should mean.  See https://github.com/zephyrproject-rtos/zephyr/pull/31722#discussion_r565719653

Seems to me it's worth getting in something that closes the high priority issue, then planning a more general solution later when we make more progress on PM.  OTOH I can just remove this, so the tests pass but anybody attempting to build samples like `hello_world` on `cpunet` will encounter a compile-time error.  Or we can keep the unintended removal of default-enabling of `CONFIG_PM` on all Nordic platforms that's been in place since 2019, but that might break out-of-tree applications. 

I think this solution's good enough for now.